File: agenceDeVoyageFront/src/app/models/voyage.ts
export class Voyage {
    private _id: number;

     private _nom: String;

     private _description: String;

     private _tarifP: String;

     private _tarifBc: String;

     private _tarifNc: String;

     private _nb: String;

    private _image: String;




public get id(): number {
return this._id;
}
public set id(value: number) {
this._id = value;
}
public get nom(): String {
return this._nom;
}
public set nom(value: String) {
this._nom = value;
}
public get description(): String {
return this._description;
}
public set description(value: String) {
this._description = value;
}
public get tarifP(): String {
return this._tarifP;
}
public set tarifP(value: String) {
this._tarifP = value;
}
public get tarifBc(): String {
return this._tarifBc;
}
public set tarifBc(value: String) {
this._tarifBc = value;
}
public get tarifNc(): String {
return 
this._tarifNc;
}
public set tarifNc(value: String) {
this._tarifNc = value;
}
public get nb(): String {
return this._nb;
}
public set nb(value: String) {
this._nb = value;
}
public get image(): String {
return this._image;
}
public set image(value: String) {
this._image = value;
}
constructor ( id: number,nom: String,description: String,tarifP: String,tarifBc: String,tarifNc: String,nb: String ,image:String ){

this.id=id;
this.tarifNc=tarifNc;
this.tarifP=tarifP;
this.tarifBc=tarifBc;
this.description=description;
this.nom=nom;
this.nb=nb;
this.image=image;
}
}

File: agencedevoyageBack/src/main/java/com/ppe/entites/Voyage.java
package com.ppe.entites;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.ManyToOne;
import javax.validation.constraints.NotNull;
@Entity
public class Voyage  {

	@Id
		@GeneratedValue
		private Long id;
		private String Nom;
		private String Description;
		private String TarifP;
		private String TarifBc;
		private String TarifNc;
		private String Nb;
        private String image;
        
        @ManyToOne
    	private FileModel filemodel;
        
		public Voyage() {
		}

		public Voyage(Long id, String nom, String description, String tarifP, String tarifBc, String tarifNc, String nb,
				String image, FileModel filemodel) {
			super();
			this.id = id;
			Nom = nom;
			Description = description;
			TarifP = tarifP;
			TarifBc = tarifBc;
			TarifNc = tarifNc;
			Nb = nb;
			this.image = image;
			this.filemodel = filemodel;
		}

		public Long getId() {
			return id;
		}

		public void setId(Long id) {
			this.id = id;
		}

		public String getNom() {
			return Nom;
		}

		public void setNom(String nom) {
			Nom = nom;
		}

		public String getDescription() {
			return Description;
		}

		public void setDescription(String description) {
			Description = description;
		}

		public String getTarifP() {
			return TarifP;
		}

		public void setTarifP(String tarifP) {
			TarifP = tarifP;
		}

		public String getTarifBc() {
			return TarifBc;
		}

		public void setTarifBc(String tarifBc) {
			TarifBc = tarifBc;
		}

		public String getTarifNc() {
			return TarifNc;
		}

		public void setTarifNc(String tarifNc) {
			TarifNc = tarifNc;
		}

		public String getNb() {
			return Nb;
		}

		public void setNb(String nb) {
			Nb = nb;
		}

		public String getImage() {
			return image;
		}

		public void setImage(String image) {
			this.image = image;
		}

		public FileModel getFilemodel() {
			return filemodel;
		}

		public void setFilemodel(FileModel filemodel) {
			this.filemodel = filemodel;
		}

		

		

		
		
        
        
        
	
	

}
